chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -1,709 +1,806 @@
+						},
+						"url": {
+							"raw": "{{url}}/rest/developers",
```

Comment:
Looks like this is failing because PUT is not supported for /developers.
I see this preset in QA. May need to check with developer if they removed this method for /developers rest end point

Reply:
yes, there's a change that was made to separate edit and merge endpoints..updates needed on both edit & merge tests, coming up in new commit

Reply:
updates pushed(worked locally)